src/Trait/CheckInsertDataTrait.php

```diff
@@ -12,7 +12,9 @@
 namespace Manticoresearch\Buddy\Trait;
 
 use Manticoresearch\Buddy\Enum\Datatype;
+//@codingStandardsIgnoreStart
```

Comment:
We should remove comments like this due to it's fixed already in the ruleset.xml, and PHPCS will not remove them when they are used only in comment blocks for type hinting